# ONT_microbiome

<br>

Under active development

Workflows for analysis of Nanopore microbiome data, created primarily for teaching.


<br>


Once you have basecalled reads (ADD IN BASECALL CODE LATER), you can assign taxonomy and get read counts using Emu. This is detailed in the `02_emu` directory [here](https://github.com/seanharrington256/ONT_microbiome/blob/main/02_emu/emu.md).



<br>
<br>
<br>
<br>
<br>
<br>
<br>



Things to pipeline out:

1. Basecalling
2. Emu (including making OTU table from output)
3. Statistics/differential abundance/diversity metrics, etc in R





